src/pomodoro.sh

```diff
@@ -268,6 +268,46 @@ function get_tag_name()
   return 0
 }
 
+# This function fetches the last pomodoro session from the database, if
+# there is one, and sets the options values accordingly.
+#
+# Return:
+# If there is a last pomodoro session, returns 0. If there isn't a last pomodoro
+# session or any of the last session arguments are invalid, returns 22 (EINVAL).
+function fetch_last_pomodoro_session()
+{
+  local last_pomodoro_session
+  local duration_in_secs
+  local tag_name
+  local description
+
+  last_pomodoro_session=$(select_from 'pomodoro_report' '"duration","tag_name","description"' '' '' 'date DESC, time DESC LIMIT 1')
+  if [[ -z "$last_pomodoro_session" ]]; then
+    options_values['ERROR']='No previous pomodoro session found'
+    return 22 # EINVAL
+  fi
+
+  duration_in_secs=$(cut --delimiter='|' --fields=1 <<< "$last_pomodoro_session")
+  is_valid_time "${duration_in_secs}s" || return "$?"
+  tag_name=$(cut --delimiter='|' --fields=2 <<< "$last_pomodoro_session")
+  is_valid_argument "$tag_name" 'tag' || return "$?"
+  description=$(cut --delimiter='|' --fields=3 <<< "$last_pomodoro_session")
+  is_valid_argument "$description" 'description' || return "$?"
+
+  options_values['TIMER']="${duration_in_secs}s"
+  options_values['TAG']=$(cut --delimiter='|' --fields=2 <<< "$last_pomodoro_session")
+  options_values['DESCRIPTION']=$(cut --delimiter='|' --fields=3 <<< "$last_pomodoro_session")
+
+  say 'Last pomodoro session:'
+  say "- Duration: ${options_values['TIMER']}"
```

Comment:
I don't think that showing the session in seconds is a user-friendly approach. You can improve this output by handling it on a correct scale. For example, if it has less than 60 seconds, use seconds; if it has less than 60 minutes, keep it under minutes, and so on.